packages/compiler/src/plugins/typescript/visitor.ts

```diff
@@ -102,6 +104,12 @@ export class Visitor {
         ),
     });
 
+    private visitFunction = (def: DeclarationReflection): ITsFunction => ({
+        ...this.makeDocEntry(def, Kind.Function),
+        inheritedFrom: def.inheritedFrom && resolveTypeString(def.inheritedFrom),
```

Comment:
what does "inheritance" mean for functions? does this callback visit class functions bound using the following arrow syntax?

```ts
class Foo {
    public bar = () => {
        ...
    }
}
```